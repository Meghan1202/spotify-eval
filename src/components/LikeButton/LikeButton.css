.Like-Button-Container {
  width: 1rem;
  height: 1rem;
}

.centered {
  position: absolute;
  transform: translate(-50%, -50%);
}
