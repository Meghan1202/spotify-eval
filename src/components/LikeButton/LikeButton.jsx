import React from 'react';
import './LikeButton.css';

const LikeButton = () => (
  <div className="Like-Button-Container">
    <img className="Heart-Button" src="./assets/red-heart.svg" alt="Heart" />
    <div className="centered">1</div>
  </div>
);

export default LikeButton;
