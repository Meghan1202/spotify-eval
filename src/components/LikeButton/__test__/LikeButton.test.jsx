import React from 'react';
import { render } from '@testing-library/react';
import LikeButton from '../LikeButton';

describe(LikeButton.name, () => {
  it('should match the snapshot', () => {
    const { container } = render(<LikeButton
      currentState
      likes={1}
      id="abc"
      patchLike={jest.fn()}
      updateSongs={jest.fn()}
    />);
    expect(container).toMatchSnapshot();
  });
});
