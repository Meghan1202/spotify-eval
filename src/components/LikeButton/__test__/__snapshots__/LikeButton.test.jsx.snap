// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LikeButton should match the snapshot 1`] = `
<div>
  <div
    class="Like-Button-Container"
  >
    <input
      alt="Like"
      class="Heart-Button"
      src="./assets/red-heart.svg"
      type="image"
    />
    <div
      class="centered"
    >
      1
    </div>
  </div>
</div>
`;
